Sources/AirRobeWidget/Config/AirRobeWidgetConfig.swift

```diff
@@ -41,3 +41,22 @@ public struct AirRobeWidgetConfig {
     }
 
 }
+
+public enum EventName: String {
```

Comment:
Let's change these values to the values in the enum in `widgets/src/events.ts:6`:
```
{
  pageview = "airrobe-pageview",
  widgetRender = "airrobe-widget-render",
  widgetNotRendered = "airrobe-widget-not-rendered",
  optIn = "airrobe-widget-opt-in",
  optOut = "airrobe-widget-opt-out",
  expand = "airrobe-widget-expand",
  collapse = "airrobe-widget-collapse",
  popupOpen = "airrobe-widget-popup-open",
  popupClose = "airrobe-widget-popup-close",
  confirmationRender = "airrobe-confirmation-render",
  confirmationClick = "airrobe-confirmation-click",
}
```